lib/buildInEnums.js

```diff
@@ -0,0 +1,167 @@
+export const predefinedActions = {
```

Comment:
```suggestion
export const actions = {
```

Reply:
Same here, what if you ever have an array of actual action objects in your code? But then this should actually also be called `predefinedActionNames` 🙈

Reply:
`import { actions as predefinedActions } from 'style-dictionary/enums';`

Exported stuff should always be somewhat brief if possible, users can quite easily make them more descriptive if they need it but the majority use case will be that brief = better and API design should be targeting majority use cases

Reply:
Alright yeah fine with me. I personally would go with a more precise naming, but most importantly we have talked about it and we have a decision. I will change that also for your other suggested renamings, where we had a similar topic.